src/scene/vcPolyModelNode.cpp

```diff
@@ -26,12 +26,31 @@ vcPolyModelNode::vcPolyModelNode(vdkProject *pProject, vdkProjectNode *pNode, vc
   OnNodeUpdate(pProgramState);
 }
 
-void vcPolyModelNode::OnNodeUpdate(vcState * /*pProgramState*/)
+void vcPolyModelNode::OnNodeUpdate(vcState *pProgramState)
 {
   const char *pTemp = nullptr;
-  vdkProjectNode_GetMetadataString(m_pNode, "culling", &pTemp, "back");
+  if (vdkProjectNode_GetMetadataString(m_pNode, "culling", &pTemp, "back") == vE_Success)
+    m_invert = !udStrEqual(pTemp, "back");
 
-  m_invert = !udStrEqual(pTemp, "back");
+  if (m_pNode->geomCount != 0)
+  {
+    udDouble3 *pPosition = nullptr;
+    udDouble3 scale;
+    udDouble3 euler;
+
+    vcProject_FetchNodeGeometryAsCartesian(pProgramState->activeProject.pProject, m_pNode, pProgramState->gis.zone, &pPosition, nullptr);
```

Comment:
Is there a possibility that `pPosition` could still be `nullptr` after this call?

Reply:
Only if there were no nodes which is checked above